agent/tests/test_ui.py

```
@@ -0,0 +1,244 @@
+        assert _resolve_static("/unknown") is None
+
+
+# ---------------------------------------------------------------------------
+# run_server
+# ---------------------------------------------------------------------------
+
+class TestRunServer:
+    @patch("iconfucius.client.server.ThreadingHTTPServer")
+    @patch("iconfucius.client.server.webbrowser")
+    @patch("iconfucius.client.server._warm_cache")
+    def test_run_server_opens_browser(self, mock_cache, mock_wb, mock_srv):
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_cache`

(ARG002)